src/overlays/actors/ovl_En_Part/z_en_part.h

```diff
@@ -5,10 +5,22 @@
 
 struct EnPart;
 
+typedef void (*EnPartActionFunc)(struct EnPart*, PlayState*);
+
+typedef enum {
+    ENPART_PARAMS_0 = 0,
+    ENPART_PARAMS_1 = 1,
+    ENPART_PARAMS_4 = 4,
+    ENPART_PARAMS_15 = 15
+} ActorParams;
+
 typedef struct EnPart {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xC];
-    /* 0x150 */ Gfx* unk_150;
+    /* 0x144 */ u8 actionFuncIndex;
+    /* 0x146 */ s16 unk146;
+    /* 0x148 */ f32 zRot;
+    /* 0x14C */ f32 unk14C;
+    /* 0x150 */ Gfx* displayList;
```

Comment:
```suggestion
    /* 0x150 */ Gfx* dList;
```
Based on a small discussion in the discord, `dList` seems the preferred term rather than the full `displayList`